src/theme/login/register.ftl

```diff
@@ -309,8 +309,11 @@
         document.getElementById('rank').value = "${(register.formData['rank']!'')}";
     }
     (function() {
-        const threshold = 250;
         let count = 0;
+        // Mobile device detection
+        const isMobile = /Mobi|Android|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
I think this is probably fine but wondering if we should align with how we detect mobile in the CSS side (via [browser width](https://github.com/defenseunicorns/uds-identity-config/blob/532-speed_robot_detection_on_mobiles_up/src/theme/login/resources/css/uds.css#L805-L806))? I'm not sure if some of the "slow robot checking" is related to the CSS/theme changes that happen on smaller screens, but seems sane to keep these aligned unless there's an issue with checking width on the JS side here.

```suggestion
        // Mobile device detection
        const isMobile = window.innerWidth < 768;
```

I think it would just be like this?